docs/includes/apiargs-MongoDBCollection-method-aggregate-option.yaml

```diff
@@ -84,11 +79,8 @@ description: |
   Indicates whether the command will request that the server provide results
   using a cursor. The default is ``true``.
 
-  For MongoDB version 2.6 or later, ``useCursor`` allows users to turn off
-  cursors if necessary to aid in replica set or shard cluster upgrades.
-
-  ``useCursor`` is ignored for MongoDB versions prior to 2.6 as aggregation
-  cursors are not available.
+  ``useCursor`` allows users to turn off cursors if necessary to aid in replica
+  shard cluster upgrades.
```

Comment:
I think we can remove this sentence entirely.

I contemplated adding a line to say that `useCursor: false` is no longer supported as of MongoDB 3.6, but it is technically still relevant for `explain` support ([PHPLIB-172](https://jira.mongodb.org/browse/PHPLIB-172), so it'd be better to keep this short and sweet with just the first sentence above.